src/main/webapp/sportchef_components/sportchef-layout-header/sportchef-layout-header.html

```diff
@@ -15,16 +15,21 @@
     You should have received a copy of the GNU Affero General Public License
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
-<link rel="import" href="../../bower_components/polymer/polymer.html" />
-<link rel="import" href="../../bower_components/paper-button/paper-button.html" />
-<link rel="import" href="../sportchef-jquery/sportchef-jquery.html" />
-<link rel="import" href="../sportchef-i18n/sportchef-i18n.html" />
-<link rel="import" href="../sportchef-imprint/sportchef-imprint.html" />
+<link rel="import" href="../../bower_components/polymer/polymer.html"/>
+<link rel="import" href="../../bower_components/paper-button/paper-button.html"/>
+<link rel="import" href="../sportchef-jquery/sportchef-jquery.html"/>
+<link rel="import" href="../sportchef-i18n/sportchef-i18n.html"/>
+<link rel="import" href="../sportchef-imprint/sportchef-imprint.html"/>
+<link rel="import" href="../sportchef-login/sportchef-login.html"/>
```

Comment:
For your next pull request it would be nice not to reformat existing, unmodified code. Your own modifications will then be easier to recognize. :-)